sdk/python/feast/cli.py

```diff
@@ -226,14 +232,16 @@ def data_source_describe(ctx: click.Context, name: str):
 
 
 @data_sources_cmd.command(name="list")
+@tagsOption
 @click.pass_context
-def data_source_list(ctx: click.Context):
+def data_source_list(ctx: click.Context, tags: list[str]):
```

Comment:
isn't this `List` instead of `list`? 🤔

Reply:
The upper case `List` has been apparently been deprecated since 3.9, the linter rightly encouraged me to use the lower case `list`.
https://docs.python.org/3/library/typing.html#deprecated-aliases

Reply:
Right, [PEP 585](https://peps.python.org/pep-0585/) introduced built-in types in Python 3.9, probably we can do some code cleanup of other useless imports as well. Thanks!

Reply:
Shuchu started doing it in #4203, but I'm sure there's a lot more left.